core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -306,6 +306,11 @@
   // Deprecated. It will be removed in 2.0.0.
   USER_UFS_FILE_WRITER_CLASS(Name.USER_UFS_FILE_WRITER_CLASS,
       "alluxio.client.netty.NettyUnderFileSystemFileWriter"),
+  USER_UFS_FILE_READ_LOCATION_POLICY(Name.USER_UFS_FILE_READ_LOCATION_POLICY,
+      "alluxio.client.file.policy.LocalFirstPolicy"),
+  USER_UFS_BLOCK_MAX_READ_CONCURRENCY_DEFAULT(Name.USER_UFS_BLOCK_MAX_READ_CONCURRENCY_DEFAULT,
```

Comment:
`USER_UFS_BLOCK_CONCURRENT_READ_MAX` to follow our convention?
you don't need `default` as every value is the default in this class.